packages/api-client/src/views/Request/ResponseSection/ResponseBodyVirtual.vue

```diff
@@ -37,7 +37,7 @@ const { mimeType, attachmentFilename, dataUrl } = useResponseBody({
       This response body is massive! Syntax highlighting won’t work here.
     </div>
     <ScalarVirtualText
-      containerClass="custom-scroll scalar-code-block border-1/2 rounded-b flex flex-1"
+      containerClass="custom-scroll scalar-code-block border-1/2 rounded-b flex flex-1 max-h-[100vh]"
```

Comment:
there's a utility class that should do it - `max-h-screen` or could use `max-h-dvh` too ([see reference](https://v3.tailwindcss.com/docs/max-height))

```suggestion
      containerClass="custom-scroll scalar-code-block border-1/2 rounded-b flex flex-1 max-h-screen"
```